src/main/java/org/osc/manager/ism/entities/DeviceMemberStatusEntity.java

```diff
@@ -0,0 +1,193 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.ism.entities;
+
+import java.util.Date;
+
+import org.osc.sdk.manager.element.DistributedApplianceInstanceElement;
+import org.osc.sdk.manager.element.ManagerDeviceMemberStatusElement;
+
+public class DeviceMemberStatusEntity implements ManagerDeviceMemberStatusElement {
+    private String version;
+    public Long rx;
+    public Long txSva;
+    public Long dropSva;
+    private String applianceIp;
+    private String applianceName;
+    private String managerIp;
+    private String brokerIp;
+    private String applianceGateway;
+    private String applianceSubnetMask;
+    private Boolean discovered;
+    private Boolean inspectionReady;
+    private String publicIp;
+    private DistributedApplianceInstanceElement distributedApplianceInstanceElement;
+
+    public DeviceMemberStatusEntity() {
+    }
+
+    public DeviceMemberStatusEntity(DeviceMemberEntity element, DistributedApplianceInstanceElement dai) {
```

Comment:
why are we storing the same information twice?

DAI has all the information and the get methods can call dai.element.getApplianceGateway() etc...

If this is because we are going to remove the DistributedApplianceInstanceElement then, please add a todo

Reply:
We actually dont need the DistributedApplianceInstanceElement  nor this class. DeviceMemberEntity itself should implement the ManagerDeviceMemberStatusElement interface.

Reply:
@emanoelxavier OK, I will implement Status Element as part of DeviceMemberEntity. DAI is not removed as it as dependency on SDK interfaces and OSC. I will add TODO and add new user story for the same.